packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -601,37 +570,12 @@ module.exports = ({ models, target }, ctx) => {
           });
         }
 
-        function formatOutput(instance) {
+        this.on('saved fetched fetched:collection', instance => {
```

Comment:
Hello @alexandrebodin, what is this  `'saved fetched fetched:collection'` mean ? (it's a curriosity)

Reply:
This is bookshelf syntax to listen on multiple events. Here we transform the output of find / findOne and after saving an entity